code/modules/admin/fun_verbs.dm

```diff
@@ -132,21 +132,24 @@
 	if(!check_rights(R_FUN))
 		return
 
-	var/input = input(usr, "This should be a message from the ship's AI.",, "") as message|null
+	var/input = input("This should be a message from the ship's AI.",, "") as message|null
 	if(!input)
 		return
 
-	if(!ai_system.Announce(input))
-		return
-
-	for(var/obj/machinery/computer/communications/C in GLOB.machines)
-		if(!(C.stat & (BROKEN|NOPOWER)))
-			var/obj/item/paper/P = new /obj/item/paper(C.loc)
-			P.name = "'[MAIN_AI_SYSTEM] Update.'"
-			P.info = input
-			P.update_icon()
-			C.messagetitle.Add("[MAIN_AI_SYSTEM] Update")
-			C.messagetext.Add(P.info)
+	if(input("Do you want to use the ship AI to say the message or a global marine announcement?",, "Ship", "Global") == "Ship")
```

Comment:
This doesn't call properly, and always returns FALSE

Reply:
Fixed